.github/workflows/linkcheck.yaml

```diff
@@ -21,11 +21,24 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r requirements.txt
       - name: LinkCheck
-        run: make linkcheck
+        id: run_tests
+        run: |
+          make linkcheck > logs.txt || echo "OUTPUT=$(grep -E "broken" logs.txt | tr '\n' '@' | sed 's/@/<br>/g' | sed 's/broken/*\*`broken`\*\*/g' | sed $'s/\e\\[[0-9;:]*[a-zA-Z]//g')" >> $GITHUB_ENV
```

Comment:
A little tiny bit of me twitches at assuming there won't be an `@` in the log line already, but in reality it should be OK, and if not we can fix it if (😢) the problem occurs.

Reply:
It is hacky, but I did not find any other better way. The main issue is the formatting from what I save to a variable to what is displayed in the GitHub body issue.

Mainly it removes colors, and try to add lines in between the commands by using html <br>.  Thanks for reviewing and yes, if it happens, we can fix it.

Reply:
I am actually glad you noticed it and hope if it happens, this comments can be helpful even for my future self. :)